src/resources/serial-issue-files.ts

```diff
@@ -0,0 +1,57 @@
+import { keys } from 'ts-transformer-keys'
+
+import { SerialIssues } from './index'
+import { DateFilter, IsInFilter } from '../filter'
+import { DatePrecision } from '../date'
+import { UnionMap } from '../util'
+
+export type Type = 'serial-issue-files'
+
+export type SerialIssueFileType = 'full-text-print-pdf'
```

Comment:
This is *very* restrictive at the moment because it's tailored pretty much to the legacy issue pdfs. It might make sense to at least add a `full-text-online-pdf` value here.

Reply:
We can add it when we need it. It should be in sync with the type we have in Postgres (which is the case right now).